### PR TITLE
[Heise] Add support for embedded youtube videos

### PR DESCRIPTION
**Pull request by kayb94**
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)

### What is the purpose of your *pull request*?
- [x] Improvement
---

### Add support for embedded youtube videos
Embedded youtube videos on heise (not jwplayer), are not being recognized right now. The two commits below add this.

### CODE REVIEW

File: youtube_dl/extractor/heise.py [hunk 1]
```diff
@@ -40,19 +54,26 @@ def _real_extract(self, url):
...
+            title = self._search_regex(
+                    r'<div[^>]+class="videoplayerjw"[^>]+data-title="([^"]+)"',
+                    webpage, 'title')
+
+        yt_videos = re.findall(
+                r'<iframe[^>]+class="yt_video"[^>]+src="//([^"]+)', webpage)
+        if yt_videos:
+            for i in range(len(yt_videos)):
+                yt_videos[i] = 'https://' + yt_videos[i] 
+            return self.playlist_from_matches(yt_videos, title, 'Youtube')
```
Comment (dstftw) [inline youtube_dl/extractor/heise.py]: This should be rewritten in terms of [`YoutubeIE._extract_urls`](https://github.com/rg3/youtube-dl/commit/66c9fa36c10860b380806b9de48f38d628289e03).

**Comment by dstftw** [comment]
Rebase on top of master so this PR does not contain changes already in master.

**Comment by kayb94** [comment]
Could you elaborate? I'm not a pro with git yet. I already did a rebase on top of youtube-dl/master, because for the changes you requested I needed your commits. I mean, at least that's what I intended to do. Is it actually something else? Maybe someone else can help out, too?

**Comment by kayb94** [comment]
I can't see the somehow duplicate commits that where there before anymore - is this ready now?

**Pull request merged**